src/components/ArtistCard.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Header from './components/Header';
+import Footer from './components/Footer';
```

Comment:
You also don't need to import Header and Footer since they are being imported in App.js